### ISSUE TITLE
💡 [FEATURE] -  Add Icons To Navbar 

### ISSUE THREAD

**Issue by sailaja-adapa** [jfmartinz/ResourceHub#1117]
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

Hello, my name is A.A.D.Sailaja. I am a Tech Enthusiastic and i'm looking forward to contribute in open source projects. Here in this project i would like to add  icons to Navbar Items. It will our website to look Attractive 

Can you please assign me this issue under gssoc'24 @jf

**Comment by jf** [comment]
> ### Idea Contribution
> 
>     * [x]  I have read all the feature request issues.
> 
>     * [x]  I'm interested in working on this issue
> 
>     * [x]  I'm part of GSSOC organization
> 
> 
> ### Explain feature request
> 
> Hello, my name is A.A.D.Sailaja. I am a Tech Enthusiastic and i'm looking forward to contribute in open source projects. Here in this project i would like to add icons to Navbar Items. It will our website to look Attractive
> 
> Can you please assign me this issue under gssoc'24 @jf
> ### Explain your solution
> 
> _No response_
> ### Any alternative approaches/features
> 
> _No response_
> ### Additional Context
> 
> _No response_

@sailaja-adapa I would suggest you pay attention to how you @ mention people. Getting such an important detail like that wrong does not look good.

@jfmartinz thoughts? you have not said anything at all through all of this.

**Comment by sailaja-adapa** [comment]
@jf i am really sorry for that

**Comment by jfmartinz** [comment]
Be careful nextime @sailaja-adapa.

**Comment by sailaja-adapa** [comment]
@jfmartinz yeah sure i am so sorry about this i am new to Contributions this won't happen again
